### ISSUE TITLE
Tabs implementation doesn't meet accessibility standarda

### ISSUE THREAD

**Issue by davidm-m** [govuk-react/govuk-react#1899]
**Describe the bug**
When navigating a page with the tab key that uses the Tabs component, you need to navigate through each tab in the list. Additionally, the left and right arrow keys do not allow you to navigate between tabs in the list.

**To Reproduce**
1. Go to the [govuk-react Tabs example](https://govuk-react.github.io/govuk-react/?path=/docs/tabs--docs)
2. Use the tab key to navigate the page
3. Observe that you cycle through each tab in the list instead of going just to the currently selected one followed by its TabPanel
4. Similarly observe that with a tab highlighted, the left and right arrow keys do not change which tab is highlighted

**Expected behavior**
According to the [WCAG example for manually activated tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/), the only tab in a tabbed interface that should be reachable when navigating a webpage using the tab key is the currently selected one. To select a different tab the left and right arrow keys can be used, followed by space or enter to confirm.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox 138.0, Chrome 135.0
(I assume it is the same in all other desktop environments, these are the only ones I have tested)

**Additional context**
The [GOV.UK Design System example](https://design-system.service.gov.uk/components/tabs/) implements this correctly. It also follows the automatic activation activation pattern but due to this WCAG guideline I think manual activation is better for React:
> Manual activation of tabs is recommended unless panels can be displayed instantly, i.e., all the panel content is present in the DOM.
